### PR TITLE
Extend docs with notification related info

### PR DESCRIPTION
**Pull request by Vodorok**
Added Detailed explanation of the feature to the online userguide.

### CODE REVIEW

File: www/userguide/userguide.md [hunk 1]
```diff
@@ -416,3 +418,17 @@ happened during a storage process under the same run name. This way you can list
...
 
 ![Run history](images/run_history.png)
+
+# <a name="userguide-server-config"></a> Server Configuration
+## <a name="userguide-notification"></a> Global Notification banner
+There is an option to set a global notification text, that will show on top of every page over the CodeChecker logo. For every connecting client. 
+
+![Notification](images/notification.png)
+
+You can edit this alert on the products page, after enabling administration buttons, with "Edit Notificiation" button, but **Super User** permissions are necessary.
```
Comment (gyorb) [inline www/userguide/userguide.md]: There is no "edit notification" button on the picture.

File: www/userguide/userguide.md [hunk 2]
```diff
@@ -416,3 +418,17 @@ happened during a storage process under the same run name. This way you can list
...
+
+![Products](images/products.png)
+
+One can disable the notification by leaving the text input empty.
```
Comment (gyorb) [inline www/userguide/userguide.md]: I would rephrase this a bit `You can remove the current notification by saving an empty text.`

Comment (Vodorok) [reply www/userguide/userguide.md]: I've modified the docs, thanks for the remarks.
